doc/contribute/contributeguide.rst

```diff
@@ -179,12 +179,27 @@ to isolate exactly which type is being referenced.
 C++ Style
 ~~~~~~~~~
 
-DAGMC conforms to the Google C++ style guide, and we have included a C++ style
-guide formatter to make developers' lives much easier. When you have added all
-the features you want to add, the style guide formatter should be run.
+DAGMC conforms to the Google C++ style guide. We use the Astyle_ code formatter
+to make developers' lives easier. Here is how to install Astyle on Ubuntu:
 ::
 
-    $ astyle --style=linux --indent=spaces=2
+    $ wget http://archive.ubuntu.com/ubuntu/pool/universe/a/astyle/astyle_3.0.1-1ubuntu1_amd64.deb
+    $ dpkg -i astyle_3.0.1-1ubuntu1_amd64.deb
```

Comment:
Does there need to be a `sudo` in front of this command?

Reply:
I guess in the documentation it should have sudo, yes. I'll fix that. I just copied those lines from the docker scripts which automatically run with sudo.

Reply:
This is ready now imo